hash_practice/exercises.py

```diff
@@ -2,19 +2,43 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(1)
     """

```

Comment:
👍  However space complexity is O(n) since you're building a dictionary of all the words.